app/pathogen/arbovirus/dashboard/filters.tsx

```diff
@@ -162,11 +202,19 @@ export default function Filters(props: { map?: mapboxgl.Map | null }) {
                 state,
                 filters.data["pathogen"],
                 map,
-                data ? data.records : [],
+                data ? data.records : []
               )}
             </div>
           )}
         </div>
+        <div>
+          <button
```

Comment:
Can we make the button sticky with the bottom of the filter bar? so that the button is always at the bottom no matter what.